site/docs/ru/guide/deployment-types.md

```diff
@@ -0,0 +1,414 @@
+| `oak`              | Oak                                                                            |
+| `serveHttp`        | `Deno.serveHttp`                                                               |
+| `std/http`         | `Deno.serve`, `std/http`, `Deno.upgradeHttp`, `Fresh`, `Ultra`, `Rutt`, `Sift` |
+| `sveltekit`        | SvelteKit                                                                      |
+| `worktop`          | Worktop                                                                        |
+
+### Ответ вебхука
+
+При получении запроса на вебхук ваш бот может вызвать до одного метода в ответе.
+Преимуществом является то, что это избавляет вашего бота от необходимости делать до одного HTTP-запроса на каждое обновление.
+Однако у этого способа есть ряд недостатков:
+
+1. Вы не сможете обработать возможные ошибки соответствующего вызова API.
+   К ним относятся ошибки ограничения скорости, поэтому вы не сможете гарантировать, что ваш запрос будет иметь какой-либо эффект.
+2. Что еще более важно, у вас также не будет доступа к объекту ответа.
+   Например, вызов `sendMessage не даст вам доступа к отправленному сообщению.
+3. Кроме того, невозможно отменить запрос.
+   Сигнал `AbortSignal` будет проигнорирован.
+4. Обратите внимание, что типы в grammY не отражают последствий выполненного обратного вызова вебхука!
+   Например, они указывают на то, что вы всегда получаете объект ответа, так что вы сами должны убедиться в том, что вы не облажаетесь при использовании этой незначительной оптимизации производительности.
+
+Если вы хотите использовать вебхук-ответы, вы можете указать опцию `canUseWebhookReply` в опции `client` вашего `BotConfig` ([документация API](/ref/core/botconfig)).
+Передайте функцию, которая определяет, использовать или нет ответ вебхука для данного запроса, идентифицированного методом.
+
+```ts
+const bot = new Bot("", {
+  client: {
+    // Мы принимаем недостаток ответов веб-хуков для ввода статуса.
+    canUseWebhookReply: (method) => method === "sendChatAction",
+  },
+});
+```
+
+Вот как работают ответы на вебхуки под капотом.
+
+```asciiart:no-line-numbers
+______________                                   _____________
+|            |                                   |           |
+|            |                                   |           |
+|            |                                   |           |
+|            |       *оба терпиливо ждут*        |           |
+|            |                                   |           |
+|  Telegram  |                                   |    Бот    |
+|            |                                   |           |
+|            |                                   |           |
+|            |   --- новое сообщение      --->   |           |
+|            |  <--- окей, sendChatAction ---    |           |
+|____________|                                   |___________|
```

Comment:
@KnorpelSenf can you launch linting and check spacing between "servers"
I tried to make them smooth